### PR TITLE
#168978212 Setting up database

### PR DESCRIPTION
**Pull request by hirwaf**
#### What does this PR do?
Having database working and change API to v2
#### Description of Task to be completed?
- changing API base from `/api/v1/` to `/api/v2/`
- install pg
- create class `database` to handle queries
- add a mock folder that contains dummy data for users and articles
- update controller to meet new structure
- add .env.example
#### How should this be manually tested?
After cloning the repo run:
- cd teamwork
- npm start
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168978212
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: server/database/database.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
```
Comment (ekumamait) [inline server/database/database.js:5]: make sure you are switching between `development and testing databases` appropriately

File: server/database/database.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      const client = await pool.connect();
+      const results = await client.query(query, params);
+      client.end();
+      return { results, errors: null };
+    } catch (errors) {
+      return { results: null, errors };
+    }
+  }
+
+  async all() {}
```
Comment (ekumamait) [inline server/database/database.js]: What is the use of this empty functions?

**Review by ekumamait — CHANGES_REQUESTED** [review]
Please configure two separate databases

File: server/database/database.js (hunk 2 above)
Comment (hirwaf) [reply server/database/database.js]: i will use them as I'm working on the features but in this RP they don't play any role

File: server/database/database.js (hunk 1 above)
Comment (hirwaf) [reply server/database/database.js:5]: yes i have

File: server/database/database.js (hunk 2 above)
Comment (ekumamait) [reply server/database/database.js]: Then I do not see any reason as to why we have them here, we could remove them until we need them

**Pull request merged**